GitTfs/Commands/Init.cs

```diff
@@ -102,6 +102,13 @@ private void DoGitInitDb()
             }
             globals.Repository = gitHelper.MakeRepository(globals.GitDir);
 
+            if (string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.name"))
+                && string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.email")))
+            {
+                globals.Repository.SetConfig("user.name", "git-tfs");
+                globals.Repository.SetConfig("user.email", "placeholder@git-tfs.com");
+            }
```

Comment:
I have tried a lot of things. None is working :(
Perhaps you could have a look in TeamCity. It seems that we could set environment variables.
Perhaps you could set this env variables in the build configuration parameters.